interview-questions.md

```diff
@@ -8,44 +8,44 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a gem?
 
-Your answer:
+Your answer: a gem is something in Ruby that is an extension. The extrension contains many files that will help Ruby to run differnt code.
 
-Researched answer:
+Researched answer: They are extensions that modify the functionality in Ruby applications. They give you access to custom methods that others wrote.
 
 2. What does the acronym CRUD stand for?
 
-Your answer:
+Your answer: CRUD is an accronym for create, read, update, delete
```

Comment:
One-line answers in an interview are not very conversational and may come off as uninterested.  This also meets the bare minimum requirements for this section.  Please make sure to be as thorough as possible in this section.